src/login.rs

```diff
@@ -504,11 +504,7 @@ impl Login {
         let client_id = self.client_id();
         do_async(
             glib::PRIORITY_DEFAULT_IDLE,
-            async move {
-                functions::RequestQrCodeAuthentication::new()
-                    .send(client_id)
-                    .await
-            },
+            async move { functions::request_qr_code_authentication(Vec::new(), client_id).await },
```

Comment:
No idea what the first argument is (other_user_ids), it isn't even nullable so we should at least get an idea for what it is.

Reply:
The response from a tdlib dev is: "Pass identifiers of other users, already logged in in the app, so they can't log in again."

@marhkb I think this is something to be handled in #166.

Reply:
Oh, nice find.